aten/src/ATen/native/Blas.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/native/Blas.h>
+#include <ATen/NamedTensorUtils.h>
+
+namespace at { namespace native {
+
+DEFINE_DISPATCH(addmv_stub);
+
+Tensor &addmv_out(Tensor& result, const Tensor &self_, const Tensor &mat, const Tensor &vec, Scalar beta, Scalar alpha) {
```

Comment:
nit: I think we should change self_ to self to keep it in sync with how the other functions are usually defined

Reply:
fixed